src/reference/workers/taskcluster-worker/engine-plugin.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Engine-Plugin interaction
+order: 40
+sequence_diagrams: true
+---
+
+Putting this all together, we get the following interactions.
```

Comment:
Which is to say, it's the tc-worker is basically undocumented from a user perspective -- that might be something for those who are hacking on it to work on!!